.github/workflows/ceph-nsfs-s3-tests.yaml

```diff
@@ -1,24 +1,31 @@
 name: NSFS Ceph S3 Tests
-on: [push, pull_request, workflow_dispatch]
+on: [workflow_call]
 
 jobs:
   nsfs-ceph-s3-tests:
     runs-on: ubuntu-latest
     timeout-minutes: 90
-    concurrency:
-      group: ${{ github.workflow }}-${{ github.ref }}
-      cancel-in-progress: true
```

Comment:
add back cancel in prog?

Reply:
added it in the workflow main level, so when a new push happens all of the workflow including all of the jobs will be cancelled